app/e-app/sand/css.html

```diff
@@ -13,7 +13,8 @@
             var content = {
                 input: '',
                 expected: undefined,
-                expectedLength: undefined
+                expectedLength: undefined,
+                qdidAttr: ''
```

Comment:
iddqd 😄 
`qdIdAttr` Perhaps?

Reply:
Just visually `qdid` looks more sexy. Not a problem to change.

Reply:
if it's more sexy, it's more sexy